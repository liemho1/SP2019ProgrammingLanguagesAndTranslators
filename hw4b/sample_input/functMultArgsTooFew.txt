{
    foo = function(x, y, z) { print(x+y+z) };
    foo(5, 4);
    print("bye")
}
