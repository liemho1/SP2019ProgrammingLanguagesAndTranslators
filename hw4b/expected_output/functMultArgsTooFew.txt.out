Line 3: Too few parameters in function call
